core-v1/src/main/scala/dsl/RunExprDsl.scala

```diff
@@ -0,0 +1,52 @@
+package com.rallyhealth.vapors.v1
+
+package dsl
+
+import algebra.CombineHolder
+import data.{ExprState, FactTable}
+
+trait RunExprDsl extends CommonDsl {
```

Comment:
Not sure I understand what you mean by this.. What type parameters would I add? What do you have to lookup in Expr.scala?

Reply:
It would be nice if there were local definitions for all the type parameters being used here or just better global documentation.  There is neither.

Said another way, if I'm trying to work in the code, I cannot just look at this file and have any sort of link back to what I am assuming is a naming convention.